tests/e2e/kcs/test_disable_mcg_external_service.py

```diff
@@ -0,0 +1,71 @@
+import pytest
+import logging
+
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs import constants
+from ocs_ci.framework.pytest_customization.marks import tier2, polarion_id, bugzilla
+
+logger = logging.getLogger(__name__)
+
+
+class TestDisableMCGExternalService:
+    @pytest.fixture()
+    def patch_noobaa_object(self, request):
+
+        # get noobaa object
+        noobaa_ocp_obj = OCP(
+            kind="noobaa",
+            namespace=constants.OPENSHIFT_STORAGE_NAMESPACE,
+            resource_name="noobaa",
+        )
+
+        # patch noobaa object
+        noobaa_ocp_obj.patch(
+            resource_name="noobaa",
+            params='{"spec": {"disableLoadBalancerService": true }}',
```

Comment:
Unless this was changed again in 4.14, the path that actually works is `/spec/multiCloudGateway/disableLoadBalancerService`.

See #7656 and #7657.

Reply:
@sagihirshfeld please check this comment by liran:
```
Hi,

We are not supporting the disableLoadBalancerService flag via the storagecluster in 4.10.
This is probably the reason its overridden.
They can re-apply the disableLoadBalancerService flag on noobaa after changing the endpoints.

We are going to provide support for the disableLoadBalancerService flag via the storagecluster in 4.13

Best Regards,
Liran.
```
So we have the support for disableLoadBalancerService flag from Noobaa CRD from 4.13 onwards.

Reply:
I agree that it is supported in 4.13, but this specific path in the YAML did not work in `test_disable_nb_lb` in 4.13, as can be seen here: https://url.corp.redhat.com/a9cb4ac.

Maybe this was changed again since in 4.13 and I'm wrong - we'll have to see for sure in the PR validation.

Reply:
IIRC it passed on 4.13